src/lib/AnnotationControlsFSM.ts

```diff
@@ -63,6 +63,9 @@ export default class AnnotationControlsFSM {
                 if (input === AnnotationInput.CREATE || input === AnnotationInput.STARTED) {
                     this.currentState = modeTempStateMap[mode] || AnnotationState.NONE;
                 }
+                if (input === AnnotationInput.CANCEL) {
+                    this.currentState = AnnotationState.NONE;
```

Comment:
Is this change needed? When we're in `NONE` state, if we don't do anything, we remain in `NONE` state, right?

Reply:
Yes that's correct, this change isn't needed.